language/json-language-server/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.0.0-PLACEHOLDER",
   "main": "src/index.ts",
   "dependencies": {
-    "@player-tools/json-language-service": "workspace:*"
+    "@player-tools/json-language-service": "workspace:*",
+    "@player-tools/static-xlrs": "workspace:*"
```

Comment:
Since we don't publish the static XLR package would we have issues listing it as a dependency here?

Reply:
I added it back by mistake. I reverted it to the [original content](https://github.com/player-ui/tools/blob/main/language/json-language-server/package.json).